Assets/Scripts/Core/AnimalController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Anivision.PlayerInteraction;
+using Anivision.Vision;
+using UnityEngine;
+
+namespace Anivision.Core
```

Comment:
Wasn't vision going to have its own namespace?

Reply:
This script is under the Core namespace because it is a script that combines the Movement and Vision parameters of each animal

Reply:
Ok, makes sense